docs/api-reference/next/image.md

```diff
@@ -147,7 +153,7 @@ When `eager`, load the image immediately.
 
 ### unoptimized
 
-When true, the source image will be served as-is instead of changing quality, size, or format. Defaults to false.
+When true, the source image will be served as-is instead of changing quality, size, or format. Defaults to `false`.
```

Comment:
If you're going to change the default here to be highlighted, also change for the other sections too.

Reply:
https://github.com/vercel/next.js/pull/20621